openhands/server/session/agent_session.py

```diff
@@ -221,6 +221,11 @@ async def _create_runtime(
             headless_mode=False,
         )
 
+        # FIXME: this sleep is a terrible hack.
+        # This is to give the websocket a second to connect, so that
+        # the status messages make it through to the frontend.
+        # We should find a better way to plumb status messages through.
+        await asyncio.sleep(1)
```

Comment:
Are we sure a second is enough time here?

Reply:
Yeah it works every time AFAICT